app/controllers/projects_controller.rb

```diff
@@ -21,6 +22,10 @@ def sort
     head :ok
   end
 
+  def toggle_archived
```

Comment:
Sorry, in my suggestion I said `toggle_archived` here but the route is actually `toggle_archive`